File: solutions/javascript/line-up/1/line-up.js
//
// This is only a SKELETON file for the 'Line Up' exercise. It's been provided as a
// convenience to get you started writing code faster.
//

export const format = (name, number) => {
  let last = number%100;
  let rank = 'th';
  if (last%10 == 1 && last != 11) {
    rank = 'st';
  }
  else if (last%10 == 2 && last != 12) {
    rank = 'nd';
  }
  else if (last%10 == 3 && last != 13) {
    rank = 'rd';
  }
  return `${name}, you are the ${number}${rank} customer we serve today. Thank you!`
};

File: solutions/python/line-up/2/line_up.py
def line_up(name, number):
    num = number%100
    rank = 'th'
    if num%10 == 1 and num != 11:
        rank = 'st'
    if num%10 == 2 and num != 12:
        rank = 'nd'
    if num%10 == 3 and num != 13:
        rank = 'rd'
    return f"{name}, you are the {number}{rank} customer we serve today. Thank you!"